packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -365,9 +365,6 @@ const config: HardhatUserConfig = {
       },
     ],
     settings: {
-      metadata: {
-        bytecodeHash: 'none',
-      },
```

Comment:
Not sure how we feel about this, for some reason it was a requirement of echidna. 
Perhaps we can set it up to only disable this setting for echidna runs.

Reply:
I'm concerned this will cause more conflicts with the `more` + bindings files. You should be able to use an env var in the hardhat config, and a `sed` command to replace the `bytecode_hash` value in the `foundry.toml` prior to running echidna.

Reply:
Fixed with this: d38cef9a9236bd2a5c0eb1b3493665192ef3f490